src/dlr.cc

```diff
@@ -258,6 +281,30 @@ extern "C" int CreateDLRModel(DLRModelHandle* handle, const char* model_path,
   API_END();
 }
 
+/*! \brief Translate c args from ctypes to std types for DLRModel ctor.
+ */
+extern "C" int CreateDLRPipeline(DLRModelHandle* handle,
+                                 int num_models, const char** model_paths,
+                                 int dev_type, int dev_id) {
+  API_BEGIN();
+  DLContext ctx;
+  ctx.device_type = static_cast<DLDeviceType>(dev_type);
+  ctx.device_id = dev_id;
+  std::vector<DLRModelPtr> dlr_models;
+  for (int i = 0; i < num_models; i++) {
```

Comment:
This logic should be implemented in CPP rather than here. Perhaps have a factory method or overloaded constructor that takes in paths vector.

Reply:
CreateDLRPipeline body is only 15 lines. It is smaller that other Create methods... Also, it needs visibility to TVMModel, RelayVMModel, TreeliteModel and HexagonModel. The only file which has it is dlr.cc.